src/Stream.tsx

```diff
@@ -15,7 +15,7 @@ let streamScript = document.querySelector<HTMLScriptElement>(
 
 function useStreamElement(
   containerRef: RefObject<HTMLDivElement>,
-  streamRef: MutableRefObject<HTMLStreamElement | undefined>
+  streamRef: MutableRefObject<HTMLStreamElement | null>
```

Comment:
`| null` conflicts with `| undefined` in the README. AFAIK, they don't actually matter since people will probably just do a truth-y test against it. Might be confusing, though.

Reply:
Ah, sorry! Good catch. We should use `null` here since that's the what the normal `RefObject` type does that people are used to when interacting with refs for DOM nodes.